packages/autorest.typescript/test/commands/test-swagger-gen.ts

```diff
@@ -907,7 +907,7 @@ const rlcTestSwaggers: { [name: string]: SwaggerConfig } = {
     isTestPackage: true,
     restLevelClient: true,
     azureSdkForJs: false,
-    generateSample: true
+    generateSample: false
```

Comment:
why we disable the sample gen here?

Reply:
Yes I disable some of sample gen in swagger side considering there will be a lot of files introduced. My temp plan is
- Only enable necessary samples in latest main 
- Fix the our browser testing issue https://github.com/Azure/autorest.typescript/issues/2015
- After that enable all samples both in swagger and typespec in our nightly build